server/controllers/annotation_controller.js

```diff
@@ -63,7 +64,7 @@ export const editAnnotation = (userId, annotationId, updateText) => {
 // Removes an annotation from the database, updates the parent's numChildren
 // and recurses if the parent needs to be removed as well.
 const deleteAnnotationHelper = (user, annotation) => {
-  if (annotation.parent === null) { // base case: annotation is top-level
+  if (annotation.parent == null) { // base case: annotation is top-level
```

Comment:
Not much difference between === and ==, but I think for consistency we want to stick with the triple?